_config.yml

```diff
@@ -1,10 +1,11 @@
 exclude: [CONTRIBUTING.md, Gemfile, Gemfile.lock, LICENSE, README.md, TERMS.md]
-permalink: /articles/:title/
+# permalink: /articles/:title/
```

Comment:
Do we want to delete this entirely vs commenting it out? Is there some reason to leave it here commented out?

Reply:
That is my understanding

Reply:
Leaving this line in, as the articles are to remain live until further notice.